### PR TITLE
[web] Add Emotion theme (example)

### PR DESCRIPTION
**Pull request by koistya**
#### Usage example:

```tsx
import { css } from "@emotion/react";

export function Example(): JSX.Element {
  return (
    <div
      css={(theme) => css`
        color: ${theme.palette.primary};
        font-family: ${theme.typography.fontFamily};
      `}
    />
  );
}
```

### CODE REVIEW

File: web/global.d.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+/**
+ * @copyright 2016-present Kriasoft (https://git.io/vMINh)
+ */
+
+import {} from "react";
+import { Theme } from "@emotion/react";
+import { Interpolation } from "@emotion/serialize";
```
Comment (Andarist) [inline web/global.d.ts:7]: isn't this type reexported from `@emotion/react`? I don't think you should depend on `@emotion/serialize`, it won't cause problems but it's really just an internal package for the most part, so people should not be expected to interact with it directly

File: web/global.d.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+    typography: {
+      fontFamily: string;
+    };
+  }
+}
+
+declare module "react" {
+  interface Attributes {
+    css?: Interpolation<Theme>;
```
Comment (Andarist) [inline web/global.d.ts:26]: Have you encountered any problem with `"@emotion/react/types/css-prop"` approach?

File: web/global.d.ts (hunk 1 above)
Comment (koistya) [reply web/global.d.ts]: Good point.

File: web/global.d.ts (hunk 2 above)
Comment (koistya) [reply web/global.d.ts]: Nope, it works. Just was experimenting with different approaches.

**Pull request merged**